tensorflow/core/kernels/matmul_op_fused.cc

```diff
@@ -102,6 +107,10 @@ struct LaunchFusedMatMulOp<CPUDevice, T> {
       case FusedComputationType::kBiasAddWithElu:
         executeWithOutputKernel(WithBiasAddAndElu<T>(bias_add_args));
         break;
+      case FusedComputationType::kBiasAddWithLeakyRelu:
+        out.device(d) = lhs.contract(rhs, dim_pair,
+                                     WithBiasAddAndLeakyRelu<T>(bias_add_args));
```

Comment:
Can we use `executeWithOutputKernel` for this?

Reply:
Yeah, the wrapper function is used